js/ts/ALGO.ts

```diff
@@ -621,7 +621,8 @@ const doQuery = async (dhead:string, query: ApiCall<any>): Promise<any> => {
   return txn;
 };
 
-export const connectAccount = async (networkAccount: NetworkAccount) => {
+const connectAccount = async (networkAccount: NetworkAccount) => {
+  const stdlibT: any = stdlib;
```

Comment:
What does the T suffix mean?

Reply:
To avoid the same name, I just appended a T for this. Which is probably a pretty poor choice. Open to name suggestions